react-toggle-switch-component/my-app/src/ToggleSwitch.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+export default function ToggleSwitch() {
+
+  const [toggle, setToggle] = useState(false);
+
+  function handleToggle()  {
+    setToggle(!toggle);
+  }
+
+ const text = !toggle ? 'Off' : 'On';
+  return (
+    <>
+      <label className="switch">
```

Comment:
Where is your CSS? Oh, you put it in App.css. I guess that's OK, though the preferred solution would be to put it in the file ToggleSwitch.css and import it here.